frontend/src/components/Search.tsx

```diff
@@ -1,67 +1,63 @@
-import { TiDeleteOutline } from 'react-icons/ti'
-import '../css/Search.css'
-import { useState, useContext } from 'react'
-import { GlobalContext } from '../context/GlobalState'
+import { TiDeleteOutline } from "react-icons/ti"
+import "../css/Search.css"
+import { useState, useContext } from "react"
+import { GlobalContext } from "../context/GlobalState"
 
 //////
 
 // import React, { useMemo, useContext } from 'react'
-import { useLoadScript } from '@react-google-maps/api'
+import { useLoadScript } from "@react-google-maps/api"
 import usePlacesAutocomplete, {
   getGeocode,
   getLatLng,
-} from 'use-places-autocomplete'
+} from "use-places-autocomplete"
 
 import {
   Combobox,
   ComboboxInput,
   ComboboxPopover,
   ComboboxList,
   ComboboxOption,
-} from '@reach/combobox'
-import '@reach/combobox/styles.css'
-import { LatLng } from './Homepage'
+} from "@reach/combobox"
+import "@reach/combobox/styles.css"
+import { LatLng } from "./Homepage"
 
 /////
 
 const Search = () => {
   const { handleSearch, setCenter, setRadius } = useContext(GlobalContext)
 
   // eslint-disable-next-line
-  const [address, setAddress] = useState('15 Broadway, Ultimo NSW 2007')
+  const [address, setAddress] = useState("15 Broadway, Ultimo NSW 2007")
   const [searchRadius, setSearchRadius] = useState(1)
   const [populationDensity, setPopulationDensity] = useState(0)
   const [timeOfDay, SetTimeOfDay] = useState(0)
   const [all, setAll] = useState(false)
   const [intersections, setIntersections] = useState(false)
   const [speedZones, setSpeedZones] = useState(false)
   const [highways, setHighways] = useState(false)
-  const [coordinates, setCoordinates] = useState<LatLng>();
+  const [coordinates, setCoordinates] = useState<LatLng>()
 
   function handleSubmit(e: React.FormEvent) {
     e.preventDefault()
 
     if (coordinates && searchRadius > 0) {
-      handleSearch(
-        {
-          latitude: coordinates?.latitude,
-          longitude: coordinates?.longitude,
-          radius: searchRadius,
-        }
-      )
-      setCenter(coordinates);
-      setRadius(searchRadius);
+      handleSearch({
+        latitude: coordinates?.latitude,
+        longitude: coordinates?.longitude,
+        radius: searchRadius,
+      })
+      setCenter(coordinates)
+      setRadius(searchRadius)
     }
-
   }
 
-
   // eslint-disable-next-line

```

Comment:
Can you see if we're using that variable? if not then might be worth removing this setState